packages/go/stbernard/workspace/golang/mod.go

```diff
@@ -61,7 +61,8 @@ func ParseModulesAbsPaths(cwd string) ([]string, error) {
 }
 
 func moduleListPackages(modPath string) ([]GoPackage, error) {
-	if result, err := cmdrunner.Run("go", []string{"list", "-json", "./..."}, modPath, environment.NewEnvironment()); err != nil {
+	if result, err := cmdrunner.Run(environment.GoCommand(),
```

Comment:
Instead of breaking lines here, let's move the list of args to a variable to make things more readable